lightbluetent/app.py

```diff
@@ -3,7 +3,10 @@
 from . import admin, home
 
 
-def create_app(config_name):
+def create_app(config_name=None):
+
+    if config_name == None:
+        config_name = "Production"
```

Comment:
Intentional capital P here?  I feel like all-lowercase is unwritten convention

Reply:
Ah, I see now, this value gets `capitalize()`d later down anyway

Reply:
Semi-intentional - it's used for a class name later (which I'd ultimately like to remove) but I have no strong feelings - you're right to note that all-lowercase is fine and used elsewhere

Happy to change, but feels legit to leave as-is too

Reply:
Ah, thanks for merging regardless!